flink-contrib/flink-storm-compatibility/flink-storm-compatibility-examples/src/main/java/org/apache/flink/stormcompatibility/wordcount/StormWordCountLocal.java

```diff
@@ -67,8 +67,10 @@ public static void main(final String[] args) throws Exception {
 
 		Utils.sleep(10 * 1000);
 
-		// TODO kill does no do anything so far
 		cluster.killTopology(topologyId);
+		// killing sends STOP signal, takes some time to clean up
+		Utils.sleep(1000);
```

Comment:
I guess that is quite easy to forget to insert a sleep after `killTopology`. Furthermore, even if you insert it and get the waiting time wrong, then your job will not have properly terminated before calling `cluster.shutdown`.